Code/Framework/AzQtComponents/AzQtComponents/Components/FancyDockingGhostWidget.cpp

```diff
@@ -76,11 +77,47 @@ namespace AzQtComponents
             window->setScreen(screen);
         }
 
-        setGeometry(targetRect);
+        QPoint midPoint = targetRect.topLeft() + QPoint(targetRect.width() / 2, targetRect.height() / 2);
+        QScreen* pointScreen = QApplication::screenAt(midPoint);
+        QRect rect(targetRect);
+
+        // On environment with multiple screens with different scaling settings, the screen coordinate system may have gaps
+        // due to the screen real estate shrinking according to the scale. When that happens, if a widget is moved into the gap
```

Comment:
I thought this was more of a discontinuity due to screens not necessarily having strict overlap in the multi-screen logical coordinate space? It might just be fine to say "the screen coordinate system may have gaps" full stop and not elaborate.

Reply:
I think Windows enforces screens to be contiguous. The issue comes when the screens are scaled, since Qt will shrink the screen coordinate system, creating gaps.
I will follow the advice to just elaborate less though, as this may actually be a broader issue on other OSs so we may as well be generic.